go/vt/sqlparser/sql.y

```diff
@@ -4843,21 +4844,37 @@ select_option:
   }
 
 select_expression_list:
-  select_expression
+  prev_pos select_expression current_pos
   {
-    $$ = SelectExprs{$1}
+    createAliasForEmpty($2, yylex.(*Tokenizer).buf[$1:$3])
+    $$ = SelectExprs{$2}
   }
-| select_expression_list ',' select_expression
+| select_expression_list ',' current_pos select_expression current_pos
   {
-    $$ = append($$, $3)
+    createAliasForEmpty($4, yylex.(*Tokenizer).buf[$3+1:$5])
+    $$ = append($$, $4)
+  }
+
+prev_pos:
+  {
+    $$ = yylex.(*Tokenizer).PrevPos
+  }
+
+current_pos:
+  {
+    $$ = yylex.(*Tokenizer).PrevPos
   }
```

Comment:
Why is this needed? It's identical to `prev_pos`.

Reply:
Fixed now. I was able to remove the `+1` business in the implementation too!